crediblee-test/index.js

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
What is all of this crediblee-test stuff?

Is it related to the project?

Reply:
Well, it was a internationalization test for a specific translation in one of the crediblee template. It's a different project with it's own package.json and when executed it shows the template with internationalization implemented.

Reply:
But overall i think is more of a test feature for crediblee instead of a logic addition to this project :)

Reply:
Do you think it should be then inside this repo? I don't see any relation with the current project.

Reply:
You right